pi-manage

```diff
@@ -793,6 +797,95 @@ def create(name, resolver):
     from privacyidea.lib.realm import set_realm
     set_realm(name, [resolver])
 
+# Event interface
+
+@event_manager.command
+def list():
+    """
+    List events
+    """
+    E = EventConfiguration()
+    events = E.events
+    print("{0:7} {4:4} {1:30}\t{2:20}\t{3}".format("Active", "Name", "Module", "Action", "ID"))
+    print(90*"=")
+    for event in events:
+        print("[{0!s:>5}] {4:4} {1:30}\t{2:20}\t{3}".format(event.get("active"),
+                                                 event.get("name")[0:30],
+                                         event.get("handlermodule"),
+                                         event.get("action"),event.get("id"),))
+
+@event_manager.command
+def enable(eid):
+    """
+    enable en event by ID
+    """
+    r = enable_event(eid)
+    print(r)
+
+@event_manager.command
+def disable(eid):
+    """
+    disable an event by ID
+    """
+    r = enable_event(eid, enable=False)
+    print(r)
+
+
+@event_manager.command
+def delete(eid):
+    """
+    delete an event by ID
+    """
+    r = delete_event(eid)
+    print(r)
+
+@event_manager.command
+def e_export(filename=None):
+    """
+    Export all events to a JSON format file
+    """
+    import pprint
+    pp = pprint.PrettyPrinter(indent=4)
+    E = EventConfiguration()
+    events = E.events
+    event_str = pp.pformat(events)
+    if filename:
+        f = open(filename, 'w')
```

Comment:
Instead of ``f = open()... f.close()``, we could use
```python
with open(filename, 'w') as f:
    f.write(event_str)
```
which saves us one line and we can be sure that the file is closed whatever happens in the with block :)

Reply:
:+1: